docs/x509.rst

```diff
@@ -182,6 +182,23 @@ X.509 Certificate Object
 
         The :class:`Name` of the subject.
 
+    .. attribute:: signature_hash_algorithm
+
+        :type: :class:`~cryptography.hazmat.primitives.hashes.HashAlgorithm`
+
+        Returns the
+        :class:`~cryptography.hazmat.primitives.hashes.HashAlgorithm` used in
+        the certificate's signature.
```

Comment:
Can we try "Returns the HashAlgorithm which was used in signing this certificate." I think with that we can drop the note. What do you think?